src/test/java/org/broadinstitute/consent/http/resources/DataRequestVoteResourceTest.java

```diff
@@ -41,7 +44,7 @@
 import static org.mockito.Mockito.when;
 import static org.mockito.MockitoAnnotations.openMocks;
 
-public class DataRequestVoteResourceTest {
+public class DataRequestVoteResourceTest implements WithLogHandler {
```

Comment:
There are a bunch of test methods that `throws Exception` but in fact do not. Look at lines 93, 105, 117, 129, 141, 152, 164, 174, and 193